backend/crawlers/ss58PrefixMigration.js

```diff
@@ -0,0 +1,187 @@
+    const batchSize = config.batchSize || 50000;
+
+    while (startBlock < endBlock) {
+        const sql = `SELECT block_number, extrinsic_index, args FROM extrinsic WHERE is_signed = TRUE AND block_number BETWEEN ${startBlock} AND ${startBlock + batchSize};`;
+        const {rows, rowCount} = await dbQuery(client, sql);
+
+        startBlock += batchSize;
+
+        if (!rowCount) {
+            logger.info(loggerOptions, `Skiped for batch ${startBlock} -> ${endBlock + batchSize}`);
+        } else {
+            logger.info(loggerOptions, `Extracted ${rowCount} rows. Batch ${startBlock} -> ${startBlock + batchSize}`);
+
+            for (row of rows) {
+                const {args, block_number, extrinsic_index} = row;
+                const nextArgs = changeAccountInJSON(args);
+                if (args !== nextArgs) {
+                    await dbQuery(client, `UPDATE extrinsic SET args='${nextArgs}' WHERE block_number=${block_number} AND extrinsic_index=${extrinsic_index};`);
+                }
+            }
+            logger.info(loggerOptions, `Finished batching for ${startBlock} -> ${endBlock + batchSize}`);
+        }
+    }
+
+    logger.info(loggerOptions, '✅ Finished migration for args property for extrinsic table');
+};
+
+const migrateSignerProperty = async (client) => {
+    logger.info(loggerOptions, 'Start migration for signer property for extrinsic table');
+
+    const result = await dbQuery(client, `SELECT DISTINCT signer FROM extrinsic WHERE signer LIKE '5%'`, loggerOptions);
+
+    logger.info(loggerOptions, `Selected ${result.rowCount} signer from extrinsic table`);
+
+    for (const row of result.rows) {
+        const {signer} = row;
+        const nextAddress = decode(signer);
+
+        logger.info(loggerOptions, `Start migration for ${signer}`);
+
+        await dbQuery(client, `UPDATE extrinsic SET signer='${nextAddress}' WHERE signer='${signer}'`, loggerOptions);
+
+        logger.info(loggerOptions, `Finished migration for ${signer}`);
+    }
+
+    logger.info(loggerOptions, '✅ Finished migration for signer property for exricnsic table');
+};
+
+const migrateBlockTable = async (client) => {
+    logger.info(loggerOptions, 'Start migration for Block table');
+
+    const result = await dbQuery(client, `SELECT DISTINCT block_author  FROM block WHERE block_author LIKE '5%'`, loggerOptions);
+
+    logger.info(loggerOptions, `Selected ${result.rowCount} accounts from block table`);
+
+    for (const row of result.rows) {
+        const {block_author} = row;
+        const nextAddress = decode(block_author);
+
+        logger.info(loggerOptions, `Start migration for ${block_author}`);
+
+        await dbQuery(client, `UPDATE block SET block_author='${nextAddress}' WHERE block_author='${block_author}'`, loggerOptions);
+
+        logger.info(loggerOptions, `Finished migration for ${block_author}`);
+    }
+
+    logger.info(loggerOptions, '✅ Finished migration for Block table');
+};
+
+const crawler = async () => {
```

Comment:
How long is migration execution? Why we can not execute it as a usual migration?

Reply:
It runs for ~ 1 hour. Let's move it to migration.